README.md

```diff
@@ -1,26 +1,73 @@
 # fastify-websocket
-
-[![Greenkeeper badge](https://badges.greenkeeper.io/fastify/fastify-websocket.svg)](https://greenkeeper.io/)
+[![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat)](http://standardjs.com/)  [![Build Status](https://travis-ci.org/fastify/fastify-websocket.svg?branch=master)](https://travis-ci.org/fastify/fastify-websocket) [![Greenkeeper badge](https://badges.greenkeeper.io/fastify/fastify-websocket.svg)](https://greenkeeper.io/)
 
 WebSocket support for [Fastify](https://github.com/fastify/fastify).
 Built upon [websocket-stream](http://npm.im/websocket-stream).
 
-## Example
+## Install
+
+```
+npm install fastify-websocket --save
+```
+
+## Usage
+
+All you need to do is to add it to you project with `register` and you are done !
+
+### Example
 
 ```js
 'use strict'
 
 const fastify = require('fastify')()
 
-fastify.register(require('fastify-websocket'), { handle })
+const options = {
```

Comment:
What do you think renaming this config as `wss` since it is set the `websocket-stream`?
I'm afraid only that this name could confuse someone like:

```
const opts = {}
const myOptions = {
   options: opts
}
```

Reply:
What about something like this ?

```js
const wssOptions = {
  verifyClient: function (info, next) {
    if (info.req.headers['x-fastify-header'] !== 'fastify is awesome !') {
      return next(false) // the connection is not allowed
    }

    next(true) // the connection is allowed
  }
}

fastify.register(require('fastify-websocket'), {
  handle,
  options: wssOptions
})
```

I think it can be less confusing for the end user, WDYT ?